src/components/FreelancerOffer/FreeOfferPopup.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import Popup from 'reactjs-popup';
+import { t } from 'i18next';
+import {
+	Actions,
+	BtnAccept,
+	BtnDecline,
+	Content,
+	Div,
+	P,
+	Header,
+	Modal,
+	P2,
+} from 'components/FreelancerOffer/FreeOfferPopup.styles';
+
+const FreelancerOfferPopup = () => {
+	const hardCode = {
```

Comment:
now this constant initialise with each rerender, its very very bad practice, move it above the component